src/Core/Services/TypeHelper.cs

```diff
@@ -304,12 +305,12 @@ public static bool TryGetDbTypeFromSqlDbDateTimeType(SqlDbType sqlDbType, [NotNu
             SyntaxKind valueKind = node.Kind;
             return valueKind switch
             {
-                SyntaxKind.IntValue => Convert.ToInt32(node.Value), // spec
-                SyntaxKind.FloatValue => Convert.ToDouble(node.Value), // spec
-                SyntaxKind.BooleanValue => Convert.ToBoolean(node.Value), // spec
-                SyntaxKind.StringValue => Convert.ToString(node.Value), // spec
+                SyntaxKind.IntValue => Convert.ToInt32(node.Value, CultureInfo.InvariantCulture), // spec
+                SyntaxKind.FloatValue => Convert.ToDouble(node.Value, CultureInfo.InvariantCulture), // spec
```

Comment:
Would be helpful to know **why** this is the correct change to make (why here?).
- Is this a breaking change? 
- does this change how values are stored in the backend database? The tests don't demonstrate this.
- examples from a chart like this would help. 

|Provided Value | client culture| culture | persisted value | returned value|
|- |- |- |- |- |
|12.32 (double)| en-DE | InvariantCulture| 12,32 | 12,32 |
|12.32 (double)| en-DE | Before change, no culture specified| 12.32| 1232??|

Reply:
In the attached issue, it is clearly mentioned that, when customer is trying to save 12.32, it is getting converted to 1232 and then same is returning back.
1) I don't think it is a breaking change. It is a fix. ideally whatever you are sending that should be saved.
2) Yes, in the issue it is already mentioned
![image](https://github.com/user-attachments/assets/009ff23f-aa73-4c07-8829-8751c79dc102)
3) it not about `en-DE` only, it about general application behavior, application should not take the decision of converting values. it should save whatever is coming from customer.